src/components/createPost/pictureList.tsx

```diff
@@ -32,6 +32,11 @@ const PictureList = () => {
     const selectedFiles = e.target.files;
     const selfilesarr = Array.from(selectedFiles);
 
+    if (selfilesarr.length + dataPics.length > 3) {
+      alert("more than 3");
```

Comment:
It just a temporary error handler right? 😄
However, variable name is difficult to read

Reply:
Also, please use variable's name in camelCase